### PR TITLE
GF-36, GF-38 - Delete and edit projects from my projects page

### PR DESCRIPTION
**Pull request by iamdavidpetric**

### CODE REVIEW

**Comment by iamalextodoran** [comment]
GF-36, GF-38 - Delete and edit projects from my projects page

Humanize the titles as much as possible.

File: src/show/pages/private/MyProjects/index.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                        label={<FaPencilAlt size='1.5rem' />}
+                        className='flex items-center justify-center w-16 h-16 text-white hover:text-green-900 hover:scale-110'
+                      />
+                    </div>
+                    <div className='flex items-center bg-red-500 w-10 h-10 rounded-full absolute -top-5 -left-4 hover:scale-110 transition-all duration-300'>
+                      <Button
+                        onClick={() => {
+                          setProjectModal(true);
+                          setProjectId(project.id);
+                        }}
```
Comment (iamalextodoran) [inline src/show/pages/private/MyProjects/index.js]: Separate func with this.

File: src/show/pages/private/MyProjects/index.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+import { FaPencilAlt, FaTrash } from 'react-icons/fa';
+
+import {
+  EDIT_PROJECT_PATH,
+  PROJECT_PATH
+} from '../../../../process/routes/paths';
+import { Button, ProgressBar, Modal } from '../../../components';
+
+const MyProjects = () => {
+  const [projectId, setProjectId] = useState();
```
Comment (iamalextodoran) [inline src/show/pages/private/MyProjects/index.js]: Initial state, null

**Review by iamalextodoran — CHANGES_REQUESTED** [review]
And check the PR title ^

**Review by MIKALAKE — APPROVED** [review]

File: src/show/pages/private/MyProjects/index.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+import {
+  EDIT_PROJECT_PATH,
+  PROJECT_PATH
+} from '../../../../process/routes/paths';
+import { Button, ProgressBar, Modal } from '../../../components';
+
+const MyProjects = () => {
+  const [projectId, setProjectId] = useState(null);
+  const [myProjects, setMyProjects] = useState([]);
+  const [projectModal, setProjectModal] = useState(false);
```
Comment (iamalextodoran) [inline src/show/pages/private/MyProjects/index.js]: Where's the reference that you delete a project through this modal?
deleteProjectModal

File: src/show/pages/private/MyProjects/index.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      .delete(`http://localhost:3000/v1/projects/${projectId}`)
+      .then(() => {
+        const filterProject = object => projectId !== object.id;
+        const filteredProjects = myProjects.filter(filterProject);
+        setProjectModal(false);
+        setMyProjects(filteredProjects);
+      })
+      .catch(err => err);
+  };
+  const getProjectId = project => {
```
Comment (iamalextodoran) [inline src/show/pages/private/MyProjects/index.js]: Empty line before this for readability and rename to something better.
The action that you do are opening a delete modal and saving the project id.

**Review by iamalextodoran — CHANGES_REQUESTED** [review]

**Review by iamalextodoran — APPROVED** [review]

**Pull request merged**